app/src/main/java/com/huji/foodtricks/foodtricks/ImageAdapter.java

```diff
@@ -21,11 +21,11 @@
             R.drawable.fish, R.drawable.flour,
             R.drawable.lemon, R.drawable.meat,
             R.drawable.mushroom, R.drawable.onion,
-            R.drawable.beans, R.drawable.butter,
```

Comment:
you can uncomment these lines, the resources we're added

Reply:
done